index.html

```diff
@@ -701,7 +701,7 @@
     </dfn> HTTP response header field can be used to communicate a policy
     indicating origin(s) that are allowed to see values of attributes that
     would have been zero due to the cross-origin restrictions. The header's
-    value is represented by the following ABNF [[!RFC5234]]:</p>
+    value is represented by the following ABNF [[!RFC5234]] (using <a href='https://tools.ietf.org/html/rfc7230#section-7'>List Extension</a>, [[!RFC7230]]):</p>

```

Comment:
but rfc 7230 only defines an extension to rfc5234. It doesn't contain the ABNF syntax. So I concluded that linking to both was better than one of them.